cmd/otelcontribcol/builder-config.yaml

```diff
@@ -50,9 +50,9 @@ extensions:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/zipkinencodingextension v0.110.0
 
 exporters:
-  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.110.0
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.110.1-0.20240927195042-40396d5fc50c
   - gomod: go.opentelemetry.io/collector/exporter/nopexporter v0.110.0
```

Comment:
Can you please update all the core modules so we have a consistent snapshot of the core repo?